contributor/es/04-mechanics-of-contributing-article-es.asciidoc

```diff
@@ -0,0 +1,158 @@
+
+Intenta examinar su documentación, los archivos de conversaciones y los artefactos de código del equipo anfitrión para desbloquearte.
+Esta situación es similar a la que usted y probablemente la mayoría de la gente se encuentra cuando utiliza uno de los proyectos populares de OSS.
+
+Al igual que en los proyectos de código abierto, pregunta al equipo anfitrión si las cosas no van a ninguna parte incluso después de intentar desbloquearte.
+Las preguntas que hagas y las respuestas que recibas ayudarán a otros que vengan después a resolver los mismos problemas.
+Asegúrate de que tu comunicación acabe en un archivo consultable que esté estrechamente vinculado al propio proyecto.
+Si ves oportunidades de mejora fáciles para alcanzar dicho objetivo si aún no se ha alcanzado, puedes intentar -muy educadamente- sugerir una mejora a tu equipo anfitrión.
+A veces el statu quo surge por pura casualidad y se mantiene así porque nadie tuvo una idea diferente o se preocupó lo suficiente.
+Las sugerencias de mejora pueden ser bienvenidas en estos casos.
+No hace ningún bien a ninguna de las partes que le des vueltas a un problema que podría resolverse en una conversación de unos minutos con alguien más informado sobre el proyecto.
+Está bien pedir ayuda.
+
+Sin embargo, hay una diferencia clave, que supone una ventaja para ti y para otras personas en el futuro:
+En casi todos los casos deberías preferir los canales de comunicación oficiales del proyecto, que pueden ser una lista de correo, una sala de chat, un gestor de incidencias o algo similar, dependiendo del propósito de tener una forma más sincrónica o asincrónica de interactuar, o de las distintas necesidades de estructuración de la comunicación.
+Todos ellos suelen tener en común que se basan en el texto, se archivan, se pueden buscar y vienen con enlaces estables: esto significa que tu pregunta y la respuesta quedarán escritas, y las referencias que enlaces en esas respuestas también se mantendrán accesibles.
+De este modo, podrá beneficiarse de estos conocimientos documentados de forma pasiva en su búsqueda Y ayudar a los futuros colaboradores a tener la misma ventaja.
+Esta documentación pasiva podría incluso servir para enriquecer la documentación "oficial", en caso de que contenga joyas especialmente valiosas, como definiciones importantes que se crearon ad hoc.
+
+Mientras trabajas, si encuentras documentación faltante (o desactualizada), hazle un favor al siguiente Colaborador y actualízala con lo que has descubierto.
+Los equipos de proyecto suelen estar encantados de recibir adiciones, actualizaciones o correcciones para su documentación existente: ¡acabas de encontrar otra oportunidad para contribuir!
+(O simplemente proporciónales amablemente un comentario sobre tu experiencia, y lo que te habría ayudado).
+
+==== Elaboración del código
+
+Todos tenemos nuestras preferencias y opiniones sobre el estilo del código, la sangría, etc.
```

Comment:
I feel like [indentación](https://es.wikipedia.org/wiki/Indentación) is more used in programming even tho its not recognized as a word according to wikipedia 🤔
```suggestion
Todos tenemos nuestras preferencias y opiniones sobre el estilo del código, la indentación, etc.
```

Reply:
I agree, I've always used `indentación` in terms of programming.

I wouldn't feel comfortable adding a _non-existent_ word, but in this case, given that it has an article on Wikipedia, I suppose it could work?

Reply:
Yeah i mean most videos i have watched they say indentacion, and also at college most teachers/students said it. And the article itself says its used in the programming community so i dont see why we shouldnt use it